src/App.tsx

```diff
@@ -1,61 +1,99 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { ToDo } from './props/TODO';
+
+const orijenTodos: ToDo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(us => todo.userId === us.id),
+  };
+});
+
+export const App: React.FC = () => {
+  const [userID, setUserID] = useState(0);
+  const [nameTask, setNameTask] = useState('');
+  const [userIDError, checkUserIDError] = useState(false);
+  const [nameTaskError, checKNameTaskError] = useState(false);
+  const [todos, addTodos] = useState([...orijenTodos]);
+
+  function chooseTaskName(e: React.ChangeEvent<HTMLInputElement>) {
+    setNameTask(e.target.value);
+    checKNameTaskError(!e.target.value);
+  }
+
+  function chooseUser(e: React.ChangeEvent<HTMLSelectElement>) {
```

Comment:
not obvious handler name, use handler naming patterns using prefix 'handl...' or 'on...'
for example: handleTitleChange, handleUserSelect, handleSubmit
and on next lines too...